scripts/console.ts

```diff
@@ -38,20 +39,38 @@ let deployments: Deployments = require(`../deployments.json`);
 // TODO: Hardware Wallet support:
 // https://docs.ethers.io/v5/api/other/hardware/
 
-const CLONE_USAGE = `USER - onboard a project to GitConsensus (create a Token & Governor)`;
-const DEV_USAGE = `DEV - deploy GitConsensus, TokenFactory, or GovernorFactory`;
+const CONTRIBUTOR_USAGE = `CONTRIBUTOR - add or check a git commit/tag on-chain, Token interactions, or Governor interactions.`;
```

Comment:
Could we make these usages `enum`s since the set of usages will probably change very little in the future (if at all)?